modal/_output.py

```diff
@@ -387,9 +392,13 @@ async def _get_logs():
                 logger.debug("App logs are done")
                 last_log_batch_entry_id = None
                 break
-            elif log_batch.image_id:
-                # Ignore image logs - these still exist for old clients
-                # New clients get them through ImageJoinStreaming (see image.py)
+            elif log_batch.image_id and not output_mgr.is_app_running:
```

Comment:
is this thing even working? i thought we don't write image logs to the app anymore

Reply:
yeah we do. Right now, that's how the web interface shows logs for image builds for the app. So there's a difference in how the web UI fetches logs vs how the CLI does.

Reply:
ok interesting, didn't know that